code/modules/mob/living/simple_animal/abnormality/teth/Brazen_Bull.dm

```diff
@@ -0,0 +1,142 @@
+/mob/living/simple_animal/hostile/abnormality/brazen_bull
+	name = "Brazen Bull"
+	desc = "A bull made of an copper and zinc alloy with someone trapped inside it"
+	icon = 'ModularTegustation/Teguicons/64x48.dmi'
+	icon_state = "Bull"
+	pixel_x = -15
+	base_pixel_x = -15
+	icon_living = "Bull"
+	maxHealth = 850
+	health = 850
+	vision_range = 11
+	aggro_vision_range = 17
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.8, WHITE_DAMAGE = 1.5, BLACK_DAMAGE = 1, PALE_DAMAGE = 2)
+	melee_damage_lower = 6
+	melee_damage_upper = 12
+	melee_damage_type = RED_DAMAGE
+	rapid_melee = 2
+	stat_attack = HARD_CRIT
+	attack_sound = 'sound/weapons/slam.ogg'
+	attack_verb_continuous = "smacks"
+	attack_verb_simple = "smack"
+	faction = list("hostile")
+	can_breach = TRUE
+	threat_level = TETH_LEVEL
+	start_qliphoth = 2
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = list(20, 20, 20, 30, 30),
+						ABNORMALITY_WORK_INSIGHT = list(40, 40, 50, 50, 50),
+						ABNORMALITY_WORK_ATTACHMENT = list(20, 25, 30, 30, 35),
+						ABNORMALITY_WORK_REPRESSION = list(50, 50, 40, 40, 40)
+						)
+	work_damage_amount = 7
+	work_damage_type = RED_DAMAGE
+	var/charge_check_time = 1 SECONDS
+	var/dash_num = 50
+	var/list/been_hit = list()
+	var/busy = FALSE
+	var/can_move = TRUE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/capote,
+		/datum/ego_datum/armor/capote
+		)
+	gift_type = /datum/ego_gifts/capote
+
+	abnormality_origin = ABNORMALITY_ORIGIN_LIMBUS
+
+/mob/living/simple_animal/hostile/abnormality/brazen_bull/FailureEffect(mob/living/carbon/human/user, work_type, pe)
+	datum_reference.qliphoth_change(-1)
+
+/mob/living/simple_animal/hostile/abnormality/brazen_bull/NeutralEffect(mob/living/carbon/human/user, work_type, pe)
+	if(prob(60))
+		datum_reference.qliphoth_change(-1)
+
+/mob/living/simple_animal/hostile/abnormality/brazen_bull/Move()
+	if(!can_move)
+		return FALSE
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/brazen_bull/Life()
+	. = ..()
+	if(!.)
+		return FALSE
+	if(!(status_flags & GODMODE))
+		if(!busy)
+			charge_check()
+
+/mob/living/simple_animal/hostile/abnormality/brazen_bull/proc/charge_check()
+	var/mob/living/carbon/human/target
```

Comment:
This variable name should be changed as "target" is an actual variable used by the mob and this could cause confusion during execution